browser-interface/packages/shared/friends/sagas.ts

```diff
@@ -644,12 +650,18 @@ function* refreshFriends() {
 
     let token = client.getAccessToken()
 
-    if (token) {
-      getUnityInstance().InitializeMatrix(token)
+    // TODO!: remove FF validation once the RPC flow is the only one
+    const isUseSocialClientEnabled: boolean = yield select(getFeatureFlagEnabled, 'use-social-client')
+    if (!isUseSocialClientEnabled) {
+      getUnityInstance().InitializeFriends(initFriendsMessage)
+
+      if (token) {
```

Comment:
This needs to be enabled when the feature flag is enabled, not the other way around

Reply:
The description of the feature flag says: "Enables the usage of the new RPC social client to consume friends instead of using the kernel implementation". So if the flag is enabled (set to true), we shouldn't notify Unity about the initialization of friends, right?